src/types.ts

```diff
@@ -66,6 +66,7 @@ export interface Result {
   close: () => void
   setEvaluations: (evaluations: Evaluation[]) => void
   registerAPIRequestMiddleware: (middleware: APIRequestMiddleware) => void
+  refreshCache: () => boolean
```

Comment:
As above, calling this `refreshCache` will confuse users. Also no need to return from it
```suggestion
  refreshEvaluations: () => void
```